src/components/ModalViews/index.tsx

```diff
@@ -62,7 +64,9 @@ export function SubmittedView({
         {children}
         {chainId && hash && (
           <ExternalLink href={getExplorerLink(chainId, hash, 'transaction')} style={{ marginLeft: '4px' }}>
-            <TYPE.subHeader>View transaction on block explorer</TYPE.subHeader>
+            <TYPE.subHeader>
+              {t('viewTransactionOn')} {t('blockExplorer')}
```

Comment:
Should we use interpolation here? I think it's better because "View Transaction on" always need context!